interfaces/builtin/network_manager.go

```diff
@@ -73,6 +73,9 @@ network packet,
 # from netplan
 /run/NetworkManager/{,**} r,
 
+# Allow writing dhcp lease files outside of the snap
```

Comment:
I understand but the wording is weird (to me anyway). Please use: `# Allow writing dhcp files to a well-known system location`